### DISCUSSION TITLE
Add Heroku Tutorial

### DISCUSSION THREAD

**Discussion by kamileecher** [AlistGo/alist#827] [Q&A]
Can you add tutorial for heroku?  Docker is different in the heroku repo you gave externally, and the docker you are currently developing is very different, its problem

**Comment by xhofe** [comment]
Do you mean `alist-heroku-postgres`?
It's temporarily unavailable, and I will add tutorial when it's available.

**Comment by kamileecher** [comment]
https://github.com/Xhofe/alist/blob/v2/Dockerfile

https://github.com/alist-org/alist-heroku/blob/v2/Dockerfile

The dockerfile in these two repos is different, how do we deploy it here?

@Xhofe

**Comment by xhofe** [comment]
https://github.com/Xhofe/alist/blob/v2/Dockerfile is universal,
https://github.com/alist-org/alist-heroku/blob/v2/Dockerfile is from https://github.com/Xhofe/alist/blob/v2/Dockerfile and it is only for heroku.
There are tutorials, they are just written in Chinese: https://github.com/alist-org/alist-heroku/blob/v2/README.md
Just click `Deploy to heroku` button, if there are something wrong, you may need to fork the repo then click in your own.

**Comment by kamileecher** [comment]
When I deploy this repo, I cannot login password.  how do i add password

https://github.com/alist-org/alist-heroku/blob/v2/Dockerfile

**Comment by xhofe** [comment]
Refer to #574 , you can get password from logs.

**Comment by kamileecher** [comment]
![Screenshot_20220327-175904_Chrome Dev](https://user-images.githubusercontent.com/91373907/160287650-057c9cc7-c82f-420e-9118-97c8bb6b2a3f.jpg)


Can you write a detailed article about it in the description of deploy heroku, because nobody knows how to do it

**Comment by xhofe** [comment]
I don't think this is necessary because it's simple enough. Just 3 steps:
- Fork the repo
- Click `Deploy to Heroku` button
- Get password from logs

**Comment by kamileecher** [comment]
Can we organize this index according to ourselves?  like adding different language or changing the design

**Reply by xhofe** [reply]
of course, you can modified the [front-end soucre code](https://github.com/alist-org/alist-web).

**Comment by kamileecher** [comment]
Thanks

Bro can you add mega nz and mail ru support

**Reply by xhofe** [reply]
Not for now, but I believe it will be soon if its api or sdk is clear enough
